client/src/pages/CalendarPage/CalendarPage.module.css

```diff
@@ -0,0 +1,53 @@
+.bg {
```

Comment:
I noticed that when hovering over a date on the calendar, it'll actually show a rectangle rather than than oval (rounded rectangle). We'll just need to apply a hover subclass to this to have the background as well as the text appropriately match the rest of the calendar.

![image](https://user-images.githubusercontent.com/24628243/156707263-c9c7031f-85c7-409f-a0b8-3c19af085de9.png)

Reply:
Changed hover style to be a rounded rectangle.